.gitignore

```diff
@@ -104,3 +209,52 @@ logs/
 
 # Project-local tools
 bin/
+
+# HTML reports
+**/*.html
+
+# ==============================================================================
+# CLAUDE AI RELATED FILES - DO NOT COMMIT
+# ==============================================================================
+.claude/
+.claude-flow/
+.swarm/
+.hive-mind/
+claude_code-gemini-mcp/
+memory/
+coordination/
+CLAUDE.md
+.mcp.json
+claude-flow.config.json
+hive-mind-prompt-*.txt
+claude-flow.ps1
+claude-flow.bat
+claude-flow.cmd
+claude-flow
+
+# Claude Flow generated files
+.claude/settings.local.json
+.mcp.json
+claude-flow.config.json
+.swarm/
+.hive-mind/
+.claude-flow/
```

Comment:
Could you please remove the changes to .gitignore from this PR since they aren't relevant to the current PR?